### DISCUSSION TITLE
Feature Requests?

### DISCUSSION THREAD

**Discussion by JayRen78** [j7126/OctoPrint-Dashboard#289] [Ideas]
Would it be possible to add the layer progress to the fullscreen dashboard?  I love this plug-in,  I keep the dashboard up and displayed on a tablet while running prints, it's exactly what I've wanted for this situation.  

I've created a Mockup of my recommendation.
![OctoPrintMockUp](https://user-images.githubusercontent.com/88173419/127590375-c394d3f4-a511-41ff-80d9-2035b3404038.jpg)

Thanks for creating this plug-in.

**Comment by 555garage** [comment]
I came here to say the exact same thing- would love to have the GCode Viewer open up in fullscreen mode.

I use this dashboard everytime I print, absoultly love the work you put in. Thank you-

**Comment by j7126** [comment]
This will be possible in v2 in the future, however I might consider adding it sooner than that.

**Comment by j7126** [comment]
#335
